doc/library/compile/function.txt

```diff
@@ -50,7 +50,7 @@ Reference
 
         Initialize attributes from arguments.
 
-.. class:: Param
```

Comment:
The list of attribute isn't in the same order and the In class have more attribute. Look at the docstring of the In class in theano/compile/io.py and its method `__init__`